test/Microsoft.IdentityModel.Protocols.SignedHttpRequest.Tests/SignedHttpRequestValidationTests.cs

```diff
@@ -1683,8 +1683,17 @@ public static TheoryData<ValidateSignedHttpRequestTheoryData> ValidateSignedHttp
                             ValidatedSignedHttpRequest = signedHttpRequest,
                         },
                         TestId = "ValidTest",
+                    },
+                    new ValidateSignedHttpRequestTheoryData
```

Comment:
Can we do one test that sets up a nonce string you expect and that validates being able to call signedHttpRequestToken.GetClaim("nonce") and is able to see the value and validate it?